### PR TITLE
Added redux-orm adapter and tests

### PR DESCRIPTION
**Pull request by carlesnunez**
Hello @aexmachina

I've been working with factory-girl with redux-orm and I'll would like to integrate the adapter with that awesome factory library.

I've made the tests and checked eslint style. We can discuss anything else if you like but this is working great for me on my actual project.

### CODE REVIEW

File: test/adapters/ReduxORMAdapterSpec.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    it('resolves to the object itself', asyncFunction(async function () {
+      const model = adapter.build('DummyReduxORMModel', {
+        id: 1,
+        type: 'City',
+        name: 'Vic',
+        country: 'ES',
+
+      });
+      const savedModel = await adapter.save(model, DummyReduxORMModel);
+      expect(savedModel).to.be.equal(savedModel);
```
Comment (aexmachina) [inline test/adapters/ReduxORMAdapterSpec.js]: This is a useless assertion.

Comment (carlesnunez) [inline test/adapters/ReduxORMAdapterSpec.js]: Should I just delete assert then?

Comment (aexmachina) [inline test/adapters/ReduxORMAdapterSpec.js]: You want to assert savedModel the same as model

**Comment by carlesnunez** [comment]
@aexmachina You are right, I removed the assert and I think that now is ready to merge :)

File: test/adapters/ReduxORMAdapterSpec.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      const model = adapter.build('DummyReduxORMModel', {
+        id: 1,
+        type: 'City',
+        name: 'Vic',
+        country: 'ES',
+
+      });
+      const savedModelP = adapter.save(model, DummyReduxORMModel);
+      expect(savedModelP.then).to.be.a('function');
+      return expect(savedModelP).to.be.eventually.fulfilled;
```
Comment (aexmachina) [inline test/adapters/ReduxORMAdapterSpec.js]: There's no reason to remove the whole test - you just want to `expect(model).to.equal(savedModel)`. However you can handle that in this test:

```js
return expect(savedModel).to.eventually.equal(model)
```

Comment (carlesnunez) [inline test/adapters/ReduxORMAdapterSpec.js]: I made the changes that you suggest me ;)

Comment (aexmachina) [inline test/adapters/ReduxORMAdapterSpec.js]: Nice one!

**Comment by aexmachina** [comment]
Thanks. Published v4.2.0

**Comment by carlesnunez** [comment]
When will be available through npm?

**Comment by aexmachina** [comment]
It's there now:

```sh
$ npm info factory-girl version
4.2.0
```

**Comment by carlesnunez** [comment]
Hello @aexmachina I was looking for dist files and I wasn't able to saw any code about reduxormadapter :S

https://github.com/aexmachina/factory-girl/blob/master/dist/index.js

The same when I tried to watch at the node package

**Comment by aexmachina** [comment]
It wasn't included in the dist files because it wasn't imported from anywhere. I added an `export ReduxORMAdapter` to index.js and published v4.2.1 👍

**Pull request merged**